internal/cmd/main.go

```diff
@@ -139,7 +139,8 @@ func InitFlags(command *cobra.Command) {
 	command.Flags().StringP("in", "i", "", "Template `string` to process (alternative to --file and --input-dir)")
 	command.Flags().String("input-dir", "", "`directory` which is examined recursively for templates (alternative to --file and --in)")
 
-	command.Flags().StringSlice("exclude", []string{}, "glob of files to not parse")
+	command.Flags().StringSlice("exclude", []string{}, "glob of files to ignore")
```

Comment:
I think the previous text was fine - "ignore" is somewhat ambiguous in this context